log4j-api/src/main/java/org/apache/logging/log4j/message/ObjectMessage.java

```diff
@@ -115,28 +108,14 @@ private boolean equalObjectsOrStrings(final Object left, final Object right) {
 
     @Override
     public int hashCode() {
-        return obj != null ? obj.hashCode() : 0;
+        return obj.hashCode();
```

Comment:
Are we certain nobody calls `new ObjectMessage(null)`?

Reply:
```
    public ObjectMessage(final Object obj) {
        this.obj = obj == null ? "null" : obj;
    }
```

Doesn't matter if they do.